documentation/library_guide/overview.rst

```diff
@@ -61,9 +61,28 @@ does (see the |dpcpp_short| specification and the SYCL specification for details
 Known Limitations
 =================
 
-For ``transform_exclusive_scan``, ``transform_inclusive_scan`` algorithms, the result of the unary operation should be
-convertible to the type of the initial value if (one is provided), otherwise it is convertible to the type of values
-in the processed data sequence: (``std::iterator_traits<IteratorType>::value_type``).
+* For ``transform_exclusive_scan``, ``transform_inclusive_scan`` algorithms, the result of the unary operation should be
+  convertible to the type of the initial value if one is provided, otherwise it is convertible to the type of values
+  in the processed data sequence: (``std::iterator_traits<IteratorType>::value_type``).
+* The definition of lambda functions used with parallel algorithms should not depend on preprocessor macros
+  that makes it different for the host and the device. Otherwise, the behavior is undefined.
+* ``exclusive_scan`` and ``transform_exclusive_scan`` algorithms may provide wrong results with
+  vector execution policies when building a program with GCC 10 and using -O0 option.
+* When used within DPC++ kernels or transferred to/from a device, a container class can only hold objects
+  whose type meets DPC++ requirements for use in kernels and for data transfer, respectively. 
+* The use of oneDPL together with the GNU C++ standard library (libstdc++) version 9 or 10 may lead to
+  compilation errors (caused by oneTBB API changes). 
+  To overcome these issues, include oneDPL header files before the standard C++ header files,
+  or disable parallel algorithms support in the standard library. 
+  For more information, please see `Intel® oneAPI Threading Building Blocks (oneTBB) Release Notes`_.
+* The ``using namespace oneapi;`` directive in a oneDPL program code may result in compilation errors
+  with some compilers including GCC 7 and earlier. Instead of this directive, explicitly use
+  ``oneapi::dpl`` namespace, or create a namespace alias. 
+* ``std::array::at`` member function cannot be used in kernels because it may throw an exception;
+  use ``std::array::operator[]`` instead.
+* Due to specifics of Microsoft* Visual C++, some standard floating-point math functions
+  (including ``std::ldexp``, ``std::frexp``, ``std::sqrt(std::complex<float>)``) require device support
+  for double precision. 
 
```

Comment:
```suggestion
When using device Unified Shared Memory (USM) allocations, one should manually copy data to the device before calling oneDPL algorithms and manually copy them back once the execution of the algorithms is finished. For more information regarding USM, please refer to SYCL™ 2020 Specification. 

```

Reply:
Thanks, Dmitriy! Does it make sense to add this as a note to buffers_and_usm page?

Reply:
Yes, this makes most sense on that special page.